connectors/fs2-connector/src/main/scala/zio/connect/fs2/FS2Connector.scala

```diff
@@ -0,0 +1,21 @@
+package zio.connect.fs2
+
+import cats.effect.kernel.Async
+import cats.effect.std.Dispatcher
+import zio._
+import zio.stream.ZStream
+
+trait FS2Connector {
+
+  def fromStream[F[_]: Dispatcher, R, A](
+    original: fs2.Stream[F, A],
+    queueSize: Int = 16
+  )(implicit
+    trace: Trace
+  ): ZStream[R, Throwable, A]
+
+  def toStream[F[_]: Async, R: Runtime, A](
+    original: ZStream[R, Throwable, A]
+  )(implicit trace: Trace): fs2.Stream[F, A]
```

Comment:
On second thought I think we should drop toStream completely. 
Users can work in connectors & ZStream land with fromStream, they will gain access to all the operators with that.

For interop heavy scenarios the interop library provides this conversion already.
I am not sure I see a place in this project for a conversion from a ZStream to an fs2.Stream.
cc @jdegoes